en/docs/chapter_searching/searching_algorithm_revisited.md

```diff
@@ -1,48 +1,48 @@
-- **Using the organizational structure of the data or the prior information contained in the data to achieve efficient element search**, such as binary search, hash search, and binary search tree search, etc.
+- **Using the organizational structure of the data or existing data to achieve efficient element searches**, such as binary search, hash search, binary search tree search, etc.
 
-It is not difficult to notice that these topics have been introduced in previous chapters, so searching algorithms are not unfamiliar to us. In this section, we will revisit searching algorithms from a more systematic perspective.
+These topics were introduced in previous chapters, so they are not unfamiliar to us. In this section, we will revisit searching algorithms from a more systematic perspective.
 
 ## Brute-force search
 
-Brute-force search locates the target element by traversing every element of the data structure.
+A Brute-force search locates the target element by traversing every element of the data structure.
 
-- "Linear search" is suitable for linear data structures such as arrays and linked lists. It starts from one end of the data structure, accesses each element one by one, until the target element is found or the other end is reached without finding the target element.
-- "Breadth-first search" and "Depth-first search" are two traversal strategies for graphs and trees. Breadth-first search starts from the initial node and searches layer by layer, accessing nodes from near to far. Depth-first search starts from the initial node, follows a path until the end, then backtracks and tries other paths until the entire data structure is traversed.
+- "Linear search" is suitable for linear data structures such as arrays and linked lists. It starts from one end of the data structure and accesses each element one by one until the target element is found or the other end is reached without finding the target element.
+- "Breadth-first search" and "Depth-first search" are two traversal strategies for graphs and trees. Breadth-first search starts from the initial node and searches layer by layer (left to right), accessing nodes from near to far. Depth-first search starts from the initial node, follows a path until the end (top to bottom), then backtracks and tries other paths until the entire data structure is traversed.
 
-The advantage of brute-force search is its simplicity and versatility, **no need for data preprocessing and the help of additional data structures**.
+The advantage of brute-force search is its simplicity and versatility, **no need for data preprocessing or the help of additional data structures**.
 
-However, **the time complexity of this type of algorithm is $O(n)$**, where $n$ is the number of elements, so the performance is poor in cases of large data volumes.
+However, **the time complexity of this type of algorithm is $O(n)$**, where $n$ is the number of elements, so the performance is poor with large data sets.
```

Comment:
> It is not related to the translation. The performance is not O ( n ) for BFS and DFS.
> 
> BTW, it should be O ( n ) not 0 ( n ) . Other places may be updated too.

There is no need to update here since it's not rendered by mathjax.js now. You can run `mkdocs serve` to preview the website. However, some configurations need to be deleted in the en/mkdocs.yml to make the command work.